gpkit/tests/t_constraints.py

```diff
@@ -49,16 +47,16 @@ def test_oper_overload(self):
         x = Variable('x')
         y = Variable('y')
         c = (y >= 1 + x**2)
-        self.assertEqual(c, 1/y + x**2/y)
+        self.assertEqual(c.posylt1_rep, 1/y + x**2/y)
```

Comment:
what is the conceptual difference between attributes called `posylt1_rep` and methods called `as_posyslt1()`?